zeppelin-web/add_dependencies.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+( sudo apt-get install libfontconfig -y || sudo yum install fontconfig -y ) || exit 0;
```

Comment:
I'm not sure it's best idea install package on the system without asking user. Shell we remove this file?